WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
LOL I thought of doing that, but tried with just "" and it didn't do anything. Then I thought it was a cheap hack, but I guess it's fine.